controller/config.go

```diff
@@ -257,5 +265,41 @@ func LoadConfig(path string) (*Config, error) {
 		}
 	}
 
```

Comment:
Experimenting with a more fluent config style, similar to the variables lib in fablab. Let me know what you think. It allows for a couple of different styes. The first is here to the next comment. This is already half the code that I initially had to configure these bits

Reply:
Here's the draft PR for the config map code: https://github.com/openziti/foundation/pull/243
It'll probably need support for a few more types and additional refinement.